packages/service-clients/end-to-end-tests/azure-client/src/test/containerCreate.spec.ts

```diff
@@ -516,5 +531,134 @@ for (const testOpts of testMatrix) {
 			const result = containerCurrent1.initialObjects.map1.get<string>("key");
 			assert.strictEqual(result, "value", "Value not found in copied container");
 		});
+
+		it("op grouping disabled as expected for 1.x clients", async () => {
+			const { container: container1 } = await clientCurrent1.createContainer(
+				schemaCurrent,
+				"1",
+			);
```

Comment:
How is this different from the test below?  I thought this would use the 1.x code (from 1-2 years ago) to load the container.

Reply:
The `clientLegacy` used below is a `1.x` version of `AzureClient`